bookings/views.py

```diff
@@ -1,3 +1,24 @@
-from django.shortcuts import render
+from django.http     import JsonResponse
+from django.views    import View
 
-# Create your views here.
+from bookings.models import Booking, Ticket
+from core.utils      import check_access, BookingStatusEnum, TicketStatusEnum
+
+class MyBookingListView(View):
+    @check_access
+    def patch(self, request):
+        user           = request.user
+        cancel_id      = int(request.GET.get('cancel'))
```

Comment:
1. 결국 받아야 되는 id의 값이 booking의 id이므로 아래와 같이 수정해 주세요.
2. patch method는 body가 존재 합니다. 따라서 쿼리 파라미터가 아닌 body로 받아 주세요.
```suggestion
        booking_id      = #body로 가져오는 코드
```

Reply:
바디로 받고 키에러에 대한 예외처리도 추가 했습니다.